rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -53,7 +53,7 @@ In addition to required header parameters as specified in RFC7515 the following
   other algorithms SHALL NOT be used.
 
 * **cty**: MUST contain the type of the transaction content indicating how to interpret it.
-* **crit** MUST contain the **sigt**, **ver** and **prevs** headers.
+* **crit** MUST contain the **sigt**, **ver**, **prevs** and **pal** headers.
```

Comment:
Not sure if `pal` should always be in `crit`, or only when the `pal` header is actually present in the JWS. [RFC7515](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.11) doesn't seem to have an issue with having it there all the time.

Reply:
What I understand from the RFC is that it MUST process it, if present. If not present, it has nothing to udnerstand about it. Or maybe if it understands it and it MUST be present, but it isn't it could reject it;

>  The "crit" (critical) Header Parameter indicates that extensions to this specification and/or [JWA] are being used that MUST be understood and processed.

Like with most standard JWS headers: your implementation MUST understand them all, but they might not be present at all times.